<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppointmentData</class>
 <widget class="QWidget" name="AppointmentData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>884</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Insert appointment data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_patient_data">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Patient data</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_edit_patient_data">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>611</width>
     <height>51</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::WidgetWidth</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_edit_sub">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>611</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_subj_exam">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Subjective examination</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_physical_exam">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Physical examination</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_edit_physical_exam">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>611</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_edit_recom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>611</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_recom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Recommendations</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_edit_drugs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>611</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_drugs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Prescribed drugs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_return">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>850</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_add">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>850</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_box_codes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>770</y>
     <width>611</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_code">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>750</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Code and issue</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_edit_reffering_unit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_ref">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Referring unit - fill if appointment is based on referral</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
